dev-requirements-py3.txt

```diff
@@ -1,7 +1,7 @@
 flake8==3.9.2
 bandit==1.7.0
 demisto-py>=3.0.2
-git+https://github.com/demisto/demisto-sdk.git@5b220445c7e84ed5e80afe83bba21b39ffd9fef7
+git+https://github.com/demisto/demisto-sdk.git@d47153f133c170d1f196a12c43e957d73c40954b
```

Comment:
This is the latest hotfix? What about the duplication flag?

Reply:
Yes.
I removed the fail duplicates flag on [demisto-sdk ](https://github.com/demisto/demisto-sdk/blob/master/.circleci/config.yml#L176) and [demisto-sdk](https://github.com/demisto/demisto-sdk/blob/master/.circleci/config.yml#L176) so this is why I changed to the lasy hotfx

Reply:
do not change the commit